src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -44,12 +44,24 @@ public void storeMessage(Message message) {
 		Entity messageEntity = new Entity("Message", message.getId().toString());
 		messageEntity.setProperty("user", message.getUser());
 		messageEntity.setProperty("text", message.getText());
-		messageEntity.setProperty("timestamp", message.getTimestamp());
+		messageEntity.setProperty("timestamp", message.gettimestamp());
```

Comment:
getTimestamp should be the correct convention

Reply:
good catch! just fixed this